charts/kargo/values.yaml

```diff
@@ -268,6 +268,16 @@ controller:
     ## @param controller.globalCredentials.namespaces List of namespaces to look for shared credentials.
     namespaces: []
 
+  committer:
```

Comment:
I doubt this is the last git client customization option we'll provide, so it may be better to name this `gitClient` because it seems like a bucket we'll put more things in in the future.

Reply:
[refactor(charts/controller): generalize committer into gitClient](https://github.com/akuity/kargo/pull/1843/commits/fc0b1556e146f07d3f141fdcdabbd5dd00c038a8)